src/create-release/index.ts

```diff
@@ -52,6 +58,23 @@ export async function actionCreateRelease() {
       }
 
       await promoteRelease(apiClient, appSlug, resolvedChannel.id, +release.sequence, releaseVersion, releaseNotes);
+
+      if (resolvedChannel.buildAirgapAutomatically) {
+        try {
+          const status = await pollForAirgapReleaseStatus(apiClient, appSlug, resolvedChannel.id, +release.sequence, "built", timeoutMinutes);
```

Comment:
This should be configurable by the end-user defining if they want to wait for the Air Gap result or not. Default should be false.

Reply:
Good idea. I have added `wait-for-airgap-build`